lib/db.js

```diff
@@ -190,7 +190,6 @@ function net_db(params, callback) {
       logger.warning('Problem contacting DITCH server: ' + err.message + "\n" + util.inspect(err.stack));
       return callback(err);
     });
-    req.end();
```

Comment:
Yes, I also tried to find the commit responsible for this but was not successful. It should be possible if you clone the old fheng/fh-mbaas-api repo but I haven't found easy way how to identify the commit.